labgrid/remote/coordinator.py

```diff
@@ -296,7 +346,12 @@ async def request_task():
                     elif kind == "startup":
                         version = in_msg.startup.version
                         name = in_msg.startup.name
-                        session = self.clients[peer] = ClientSession(self, peer, name, out_msg_queue, version)
+                        client_session = self._session_name(peer, in_msg.startup.session)
+                        session = self.clients[peer] = ClientSession(
+                            self, peer, name, out_msg_queue, version, client_session
+                        )
+                        if session.session:
+                            self.session_manager.add_session(client_session)
```

Comment:
I think some variable names here are a bit confusing. I'd prefer something like this:
```suggestion
                        id = self._session_id(peer, in_msg.startup.session)
                        session = self.clients[peer] = ClientSession(
                            self, peer, name, out_msg_queue, version, id
                        )
                        if session.id:
                            self.session_manager.add_session(id)
```

Reply:
I agree that looks better, done.